modules/misc/test/tests/kernels/diffusion/tests

```diff
@@ -23,7 +23,7 @@
     run_sim = True
     petsc_version = '>=3.9'
     method = 'OPT'
-    cli_args = 'Executioner/num_steps=5'
+    cli_args = 'Executioner/num_steps=5 Outputs/exodus=false'
```

Comment:
whoa...really? This seems like it should throw an error before allowing this?

Reply:
Or suppress all outputs. @dschwen 's script suppressed, but it always caused run time errors if I was trying to refer to a named output block.

Reply:
> whoa...really? This seems like it should throw an error before allowing this?

What should throw an error about what? Are you saying you'd like to us to by default suppress all output when running the `PetscJacobianTester`? That would be a fair request

Reply:
I see that the remaining failures are the same issue

Reply:
I'm assuming from your commit text, there is interference between outputs if the jacobian tester uses the same input file as a Exodiff or Csvdiff. If this were to happen for two Exodiffs on the same output, it throws an error (I think), but since this is a jacobian test, it doesn't check that condition. Since the outputs aren't being used, they should probably just be suppressed, albeit without removing the named output blocks for other objects to output to.

Reply:
Yea most of the time you don't want to use the output, but you can actually make use of it in a testing situation, e.g.:
```
  [./stateful_jacobian]
    type = 'PetscJacobianTester'
    input = 'ad_stateful_material.i'
    ratio_tol = 1e-7
    difference_tol = 1e-5
    run_sim = True
    petsc_version = '>=3.9'
    requirement = 'AD shall work with stateful material properties and the Jacobian shall be beautiful'
    design = "jacobian_definition.md"
    issues = "#5658"
  [../]
  [./stateful_residual]
    type = 'Exodiff'
    exodiff = 'ad_stateful_material_exodus.e'
    input = 'ad_stateful_material.i'
    should_execute = false
    delete_output_before_running = false
    prereq = 'stateful_jacobian'
    requirement = 'The time evolution of the stateful diffusivity property shall show that AD works with stateful material properties'
    design = "jacobian_definition.md"
    issues = "#5658"
  [../]
```
But I agree that we can disable the outputs by default

Reply:
Yeah, I guess that can save you time, but that becomes pretty intertwined